model_signing/pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.ruff]
+line-length = 80
+target-version = "py311"
+
+[tool.ruff.lint]
+select = ["B", "E", "F", "I", "N", "PLC", "PLE", "SIM", "W"]
+ignore = ["B024"]
```

Comment:
We'll probably fix this for later, make sure all abstract classes have methods marked as abstract. But not for this PR

Reply:
Sure. Specifically it took issue with the empty class here:
https://github.com/sigstore/model-transparency/blob/90cac6c4634300a6cbe54f4cdd011290dc1062dd/model_signing/manifest/manifest.py#L128-L140

Reply:
Oh, I assumed all of empty means trivially true, but it seems they want at least one method. Well, this will get fixed soon anyway